payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SaveForFutureUseElementUI.kt

```diff
@@ -63,12 +61,11 @@ fun SaveForFutureUseElementUI(
             onCheckedChange = null, // needs to be null for accessibility on row click to work
             enabled = enabled
         )
-        Text(
-            resources.getString(controller.label, element.merchantName),
-            Modifier
+        H6Text(
+            text = resources.getString(controller.label, element.merchantName),
+            modifier = Modifier
```

Comment:
Do we have any plans for RTL text features? If so, I think padding start doesn't make sense here

Reply:
For now we don't have any RTL languages. So we should be good to keep this. Lots of places in our code base need to be fixed, so for now I think it's okay since it isn't planned. 

Thought isn't padding start okay because it would start from the correct side when switched? I thought it was preferred over using padding left. I might be wrong.

Reply:
Oh yea true, I was thinking start = left, but I think you are right.